### ISSUE TITLE
Better suggestion tracker

### ISSUE THREAD

**Issue by dfabulich** [iftechfoundation/ifdb-suggestion-tracker#100]
We've got a lot of suggestions here; Github isn't that great for sorting through them, allowing users to vote on them, etc.

Apropos #88, we could implement our own, or, more likely, switch to a better suggestion tracker. (Do people like UserVoice?)

**Comment by dfabulich** [comment]
One feature that would be really nice would be SSO, so you don't have to create a new account on a new website to participate in the suggestion tracker.

**Comment by curiousdannii** [comment]
You can sort reactions actually: https://github.com/iftechfoundation/ifdb-suggestion-tracker/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by dfabulich** [comment]
I agree, but I stand by my claim that it "isn't great." I think most people don't realize that reactions can act as votes, so they aren't reacting/"voting." UserVoice makes that clearer, and puts a limit on the number of votes per user, making the votes more meaningful.

**Comment by qdacsvx** [comment]
Single-sign-in for both websites would be convenient. Would users be able to comment on Uservoice anonymously or pseudonymously? If I login with my IFDB account what if I want to make an unpopular suggestion like "ban people for X". I would fear payback on IFDB against my account.

I don't know how much take up there will be. Many people come to IFDB for entertainment or just to read reviews. Someone in a groove won't necessarily notice a new link for suggestions on IFDB.

**Comment by qdacsvx** [comment]
BTW I often don't view the homepage due to lag so I wouldn't even see a new link if it was only there.

We have too many poor quality suggestions with no description, no context (why does this help? what problem does it solve?) no pros/cons, no alternatives, no user story, no ideas for implementation, no practical issues, no links to required/motivating suggestions... Perhaps there could be a form for making suggestions with prompting questions on it to help people make better suggestions.

There isn't a culture of asking follow up questions. Often a suggester (or commenter) needs help to explain and can do a better job in a discussion than in a single suggestion or comment. There is a role for someone not focussed on making suggestions but just on bringing out contributors - a suggestions helper.

Discussion often meanders off topic with little discussion of the suggestion.

Suggestions sometimes appear in comments. Should these be promoted to the list?

Some suggesters find it difficult to bite the bullet and say what they mean. It's easier to write 10 ideas than to make suggestions - hence suggestion request thread #80 pinned and always available.

Voting is slow even if you know how. There could be a READ-ME-FIRST pinned explaining how. Using emotional reactions to vote is messy compared to a simple vote interface.

I understand Uservoice has a voting gadget for each item in the list of topics? That's faster. While a few people may vote ignorantly, after several up votes, a topic becomes much more interesting to the public and will get attention. Topics which are downvoted (at the bottom) may get attention from people looking to diagnose/fix them.

**Comment by qdacsvx** [comment]
Categories would help to provide context for suggestions and prompt people to think of bugs and new suggestions.
Categories corresponding to user actions e.g. categories on IFDB personal page. Cats could move up and down depending on popularity. Also (not shown) for pages of the web site. Users should be able to create new (sub-) categories for suggestions in new topics. Option to follow cats of interest. Following a cat follows sub-cats. There should be a way to see all new comments.

Reviews
 reading
 writing
 voting
 comments on reviews
 review sorting
 spoilers
 moderation

Discussions
 finding
 commenting
 reading
 likes (suggested)
 spoilers
 moderation

Clubs
 creating
 joining
 admin

Competitions

Interactive Fiction
 finding
 download
 play online
 website (link)
 rating
 x-link reviews
 x-link commenting
 publishing
  creating a new page
  editing page
 tags
 spoilers
 content warnings
 moderation

Recommended Lists

Polls
 voting
 sharing

Lists

Play Lists
 sharing
 automation (suggested)

Profiles
 find member's profile
 edit

User Settings
 find
 settings wanted
 useless settings

Misc
 frequent fiction points
 notifications (suggested)

Moderation

Thinking outside the Box

Off topic

Introduction - say hello

**Comment by alice-blue** [comment]
Regardless of which suggestion tracker you use, I suggest explaining (on the suggestion tracker itself) how it's expected to be used. If people see a problem with IFDB but need discussion/feedback before proposing a solution, where are they supposed to do that (on the suggestion tracker, or somewhere else)? How much detail are people supposed to go into? etc. Some people may be more familiar with forum discussions than with issue trackers and not know what the rules are.

**Comment by qdacsvx** [comment]
An exploded view of categories: sprinkle "Make a suggestion" links all over IFDB.

**Comment by qdacsvx** [comment]
To avoid the existing suggestions getting too many votes for new suggestions to compete, the existing suggestions should be added to the new tracker gradually, 5-10 per day; or 1-2 per category.
